### PR TITLE
bt-301: Fix title for ExperienceYears on slider

### LINKED ISSUE #301: Fix title for ExperienceYears on slider

### LINKED ISSUE #313: [BUG] Not depending on the placement of the slider name of the “Experience Level” is "Beginner" on the screen of step 01/05
Description:
[Mobile]: Not depending on the placement of the slider name of the “Experience Level” is "Beginner" on the screen of step 01/05

Environment:
 OS - Android
 Version - 12 SP1A.210812.016

Severity Minor (S4)

Steps to reproduce:

1. Open the app
2. Sign Up with valid data, press “Sign Up”
3. After the redirecting move the slider of the “Experience Level”
4. Pay attention to the slider name

Actual Result:
Not depending on the placement of the slider name of the “Experience Level” is "Beginner" on the screen of step 01/05

Expected Result:
Year of Experience should be a range input with values from No(0 years) to 5+ years with a step of a 0,5 year
slider

### PR DESCRIPTION
**Pull request by anogire**
Screenshot 2023-09-07 at 15 14 39
Screenshot 2023-09-07 at 15 14 34
Screenshot 2023-09-07 at 15 14 25

### CODE REVIEW

File: mobile/src/bundles/common/helpers/get-years-title/get-years-title.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+const getYearsTitle = (value?: number): string => {
+    const singleYear = 1;
+    return value ? `${value} year${value === singleYear ? '' : 's'}` : 'no';
```
Comment (nikita-remeslov) [inline mobile/src/bundles/common/helpers/get-years-title/get-years-title.ts]: Conditional operator in Conditional operator. Try to write few unit test for it)

Comment (nikita-remeslov) [reply mobile/src/bundles/common/helpers/get-years-title/get-years-title.ts]: we can simplify it

**Review by nikita-remeslov — CHANGES_REQUESTED** [review]

**Comment by katerynakostikova** [comment]
I'd better pass value to selector like this, value and label array of objects 
The label can be undefined and if yes, we just show value, if there is a label we'll show it 
In that way label can be whatever you want

**Comment by anogire** [comment]
Screenshot 2023-09-08 at 20 23 56
Screenshot 2023-09-08 at 20 24 02
Screenshot 2023-09-08 at 20 24 10
Screenshot 2023-09-08 at 20 24 16
Screenshot 2023-09-08 at 20 24 21
Screenshot 2023-09-08 at 20 24 25

But, if there is no label (in some other case) It will look like this. I could not move it more closely. :(
Screenshot 2023-09-08 at 20 27 03
Screenshot 2023-09-08 at 20 27 10

**Review by katerynakostikova — APPROVED** [review]

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**